.github/workflows/memory-tests.yml

```diff
@@ -34,5 +34,5 @@ jobs:
         working-directory: test-packages/memory-tests/sdk-canary/
       - run: node --inspect test-script.js && cat result.txt
         working-directory: test-packages/memory-tests/sdk-canary/
-      - run: yarn ts-node scripts/compare-memory-usage.ts $(cat ./test-packages/memory-tests/sdk-v1/result.txt) $(cat ./test-packages/memory-tests/sdk-canary/result.txt)
-        name: compare v1 and canary
+      - run: yarn ts-node scripts/compare-memory-usage.ts $(cat ./test-packages/memory-tests/sdk-v2/result.txt) $(cat ./test-packages/memory-tests/sdk-canary/result.txt)
+        name: compare v2 and canary
```

Comment:
[q] Instead of `v2`, should we use something like `previous` so we don't have to maintain this in the future? Same for variable names in `compare-memory-usage.ts`

Reply:
I think we have to maintain this anyway because of breaking changes and I feel that 'compared to v2' gives us a better understanding than 'compared to previous'. This part is a simple search and replace, I'd like to keep it because I think the added clarity of what we compare is worth the effort of this search/replace.